go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/pokt-network/pocket
+
+go 1.17
```

Comment:
Any chance we can start of with 1.18. I was hoping to use generics!

Reply:
1.18 is still beta; I was hoping to migrate to it when it goes live.

Reply:
This is the kind of reasoning that should live in documentation.

Reply:
I also think that adding a comment in the `go.mod` file above `go. 1.17` will make it very clear to the reader (without needing to look in a different place) why we're not on 1.18 yet.